### PR TITLE
first time app is loaded, should show welcome page with example datasets

### PR DESCRIPTION
**Pull request by karissa**
Welcome page for 1.0 id

### CODE REVIEW

File: browser/sync_usb.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+var osmServerHost = remote.getGlobal('osmServerHost')
 var ws = wsock('ws://' + osmServerHost)
 
-var log = require('../lib/log').Browser()
-
 var replicationProgress = 0
+var file
 
 pump(ws, split(JSON.parse), through.obj(function (row, enc, next) {
+  console.log(row)
```
Comment (gmaclennan) [inline browser/sync_usb.js]: Remove before merge?

File: browser/sync_usb.js [hunk 2]
```diff
@@ -41,33 +44,29 @@ pump(ws, split(JSON.parse), through.obj(function (row, enc, next) {
...
 
 var resdiv = document.getElementById('response')
 var cancelBtn = document.getElementById('cancel')
 var selectExistingBtn = document.getElementById('select-existing')
 var selectNewBtn = document.getElementById('select-new')
 var sourceField = document.querySelector('form#sync input[name="source"]')
 
-ipc.on('select-file', function (event, file) {
-  if (!file) return
-  sourceField.value = file
```
Comment (gmaclennan) [inline browser/sync_usb.js]: I think this was to show to the user the file they had selected? Do you think we should remove this?

File: package.json [hunk 3]
```diff
@@ -72,7 +73,8 @@
     "osm-p2p-api": "^1.0.1",
     "osm-p2p-defork": "^1.0.0",
     "osm-p2p-geojson": "^3.0.0",
-    "osm-p2p-server": "^2.3.1",
+    "osm-p2p-server": "^2.3.5",
+    "pretty-bytes": "^4.0.2",
```
Comment (gmaclennan) [inline package.json]: Is this used anywhere?

File: package.json [hunk 4]
```diff
@@ -61,6 +61,7 @@
     "end-of-stream": "^1.1.0",
     "hyperlog": "^4.12.0",
     "hyperlog-sneakernet-replicator": "^1.1.2",
+    "inherits": "^2.0.3",
```
Comment (gmaclennan) [inline package.json]: Is this used anywhere?

File: browser/sync_usb.js (hunk 2 above)
Comment (karissa) [reply browser/sync_usb.js]: It seems like that input box has been hidden in a previous PR. I agree it'd be a good idea to show the user the file they had selected, though.

File: package.json (hunk 3 above)
Comment (karissa) [reply package.json]: good catch!

File: browser/sync_usb.js (hunk 1 above)
Comment (karissa) [reply browser/sync_usb.js]: done

**Pull request merged**